src/EFCore.PG.CockroachDB/EFCore.PG.CockroachDB.csproj

```diff
@@ -6,6 +6,9 @@
         <Nullable>enable</Nullable>
         <AssemblyName>Npgsql.EntityFrameworkCore.CockroachDB</AssemblyName>
         <RootNamespace>Npgsql.EntityFrameworkCore.CockroachDB</RootNamespace>
+        <IsPackable>true</IsPackable>
+        <PackageId>CockroachDB.EFCore.Provider</PackageId>
+        <Version>0.0.1</Version>
```

Comment:
Does this versioning scheme make sense? I know its going to be pre-release thing, but probably want to pad this out so that you can publish fixes easily

Reply:
How do you propose we pad this out?

Reply:
I was following https://semver.org/ major.minor.patch

Reply:
This is now 0.1.0